packages/itwinui-css/src/button/split-button.scss

```diff
@@ -3,43 +3,54 @@
 @import '../style/index';
 
 @mixin iui-button-split {
-  @include iui-button-group;
+  display: inline-flex;
+  align-items: center;
+  isolation: isolate;
 
-  &:hover {
-    background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+  :where(.iui-button:first-child) {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+
+    &[data-iui-variant='high-visibility'],
+    &[data-iui-variant='cta'] {
+      border-right-color: rgba(var(--iui-color-foreground-accessory-rgb), var(--iui-opacity-4));
```

Comment:
Can we use a different value for this part? We won't be able to do this with the new variables package.

Reply:
> that variable combo is the same used in [default.scss border](https://github.com/iTwin/iTwinUI/blob/0425f911ba23869f341421de7d948cc667b48a3d/packages/itwinui-css/src/button/default.scss#L9).

no, that is using `foreground-body`, this is using `foreground-accessory`, for which we don't have a scale. (see [current variables list](https://unpkg.com/@itwin/itwinui-variables))

If we are ok with the visuals in `background-primary-hover`/`background-positive-hover`, then that should be fine.

Reply:
I'm ok with it for now, I want to update the base light & dark theme to meet WCAG AA anyways and this should be improved then.

Reply:
I like white/bight one more 😕

Reply:
Ok, lets revert to use our original color for this PR & we will address the new color in #718.

Reply:
It's also gonna be blocking #718 so we need to have this conversation one way or another.

Reply:
Actually I think we can just hardcode `0, 0, 0`, there is no need for a new color variable.

Reply:
That makes sense to me, in #718 we can hard code a color since the value does not change from theme to theme.

Reply:
Can we do it in this PR? I don't want to deal with merge conflicts again.

Reply:
Ok, lines 18 & 42 are now using `--_iui-button-split-divider-color` which is set to `rgba(255, 255, 255, var(--iui-opacity-4));` which results in no visual change from what is live right now.

Reply:
there is no need for a css variable here, i would either make it a sass variable or hardcode it. css variables should be for values that need to change later

Reply:
Replaced with a SASS variable.